modules/cnv-edit-cdrom-vm.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-edit-cdrom_{context}"]
+
+= Editing CD-ROMs for virtual machines
+
+.Procedure
+
+. From the *{object-gui}* tab, select your {object}.
+. Select the *Overview* tab.
+. Click the pencil icon next to the *CD-ROMs* label to open the *Edit CD-ROM* window.
+* If no CD-ROMs are available for editing, the CD label initially displays *blank*.
+* If there are CD-ROMs avaiable, you can eject the CD-ROM by clicking *Eject CD-ROM* or remove it by clicking *-*.
+. In the Edit CD-ROM window, do the following:
+.. Select the type of CD-ROM configuration in the *Media Type* field.
+.. Complete the required information for each *Media Type*.
+.. Click *Add CD-ROM*.
+* On Windows systems, *Windows guest tools* is attached by default in
```

Comment:
Clarification: Is this bullet supposed to go under "Click *Add CD-ROM*" or one of the other *Media Type* steps?

Reply:
Yeah, I think it's better under the first sub-bullet as you first notice this on Windows when the Edit dialog opens. So....

.. Select the type of CD-ROM configuration in the *Media Type* field.
* On Windows systems, *Windows guest tools* is attached by default in
the *Media Type* field.

I think it's better on the first because the default Windows behavior overrides this task step. But let me know if you think the second sub-bullet is a better place for it.